### PR TITLE
optimize : enhance fail/timeout status metric and log level

### PR DESCRIPTION
**Pull request by Bughue**
- [ ] I have registered the PR [changes](https://github.com/seata/seata/tree/develop/changes).

### Ⅰ. Describe what this PR did
1. 对fail做了summary和timer上报
2. 对2阶段timeout做了summary上报（不需要timer，因为超时都是一个时间）
3. fix defaultcoodinator的日志级别

### Ⅱ. Does this pull request fix one issue?

https://github.com/seata/seata/issues/4516

### CODE REVIEW

**Review by slievrly — APPROVED** [review]
LGTM

**Comment by slievrly** [comment]
@Bughue add commit log in change/1.5.0 zh-cn/en

File: metrics/seata-metrics-api/src/main/java/io/seata/metrics/IdConstants.java [hunk 1]
```diff
@@ -66,4 +66,9 @@
     String STATUS_VALUE_COMMITTED = "committed";
 
     String STATUS_VALUE_ROLLBACKED = "rollbacked";
+
+    String STATUS_VALUE_FAILED = "failed";
+
+    String STATUS_VALUE_TWO_PHASE_TIMEOUT = "2phase.timeout";
```
Comment (slievrly) [inline metrics/seata-metrics-api/src/main/java/io/seata/metrics/IdConstants.java:72]: prometheus does not recognize dots.

**Comment by Bughue** [comment]
> @Bughue add commit log in change/1.5.0 zh-cn/en

ok

**Pull request merged**